pkg/vadmin/describe_db_at.go

```diff
@@ -42,7 +42,7 @@ func (a Admintools) DescribeDB(ctx context.Context, opts ...describedb.Option) (
 
 // genDescribeCmd will generate the command line options for calling
 // admintools -t revive_db --display-only.
-func (a Admintools) genDescribeCmd(s *describedb.Parms) []string {
+func genDescribeCmd(s *describedb.Parms) []string {
```

Comment:
I turned the method into a simple function because it did not have any dependencies on `Admintools`. I saw that in many cases you used receiver functions even though we are not using any fields from `Admintools`. Any reasons why?

Reply:
I like to keep the admintools commands together under the same struct. It makes it obvious that we are generating a command for admintools. I think functionally it would be fine to drop the receiver. But I think that makes sense only if you find a case where you can reuse the function outside of an `Admintools` function.